cmd/debugJs.go

```diff
@@ -0,0 +1,88 @@
+package cmd
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"encoding/json"
+
+	"os"
+
+	"github.com/StackExchange/dnscontrol/models"
+	"github.com/StackExchange/dnscontrol/pkg/js"
+	"github.com/urfave/cli"
+)
+
+func exit(err error) error {
+	if err == nil {
+		return nil
+	}
+	return cli.NewExitError(err, 1)
+}
+
+var debugJSCommand = &cli.Command{
+	Name:  "debug-js",
```

Comment:
s/debug-js/debug-otto/g

Reply:
I'd rather not mention otto in the command. There are other js interpreters I may eveluate and try to plug in in the future. otto is just one implementation.